autowarefoundation/autoware
# blinker control added

## Status
**PRODUCTION / DEVELOPMENT**

## Description
This adds a parameter which allows to choose the no.of waypoints before which the blinkers should turn ON. 

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()


## Todos


## Steps to Test or Reproduce

launch eveything and set this `str_wp_ahead_of_curvature` in Decision Maker to the no.of waypoints before which the blinkers should turn ON. and set `target_waypoint` in descion maker to 1.
The blinkers should turn on ahead of those many waypoints.
